app/decorators/govspeak_decorator.rb

```diff
@@ -6,13 +6,25 @@ def self.translate_markdown(markdown, sanitize: true)
     newdoc.to_html
   end
 
-  Govspeak::Document.extension('YoutubeVideo', /\$YoutubeVideo(?:\[(.*?)\])?\((.*?)\)\$EndYoutubeVideo/m) do |title, youtube_link|
-    youtube_id = youtube_link.scan(/^.*(youtu.be\/|v\/|u\/\w\/|embed\/|watch\?v=|&v=)([^#&?]*).*/)[0][1]
+  Govspeak::Document.extension('YoutubeVideo', /\$YOUTUBE(?:\[(.*?)\])?\((.*?)\)\$YOUTUBE/m) do |title, youtube_id|
     embed_url = %(https://www.youtube.com/embed/#{youtube_id}?enablejsapi=1&amp;origin=#{ENV['DOMAIN']})
-    optional_title = title ? %(title="#{title}") : ''
-    %(<div class="govspeak-embed-container"><iframe class="govspeak-embed-video" src="#{embed_url}" #{optional_title} frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>)
+    optional_title = title ? "Video: #{title}" : ''
+    %(<h2 class="govuk-heading-m govuk-!-margin-top-2">#{optional_title}</h2><div class="govspeak-embed-container" style="padding:56.19% 0 0 0;position:relative;"><iframe class="govspeak-embed-video" style="position:absolute;top:0;left:0;width:100%;height:100%;" src="#{embed_url}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>)
   end
 
+  Govspeak::Document.extension('VimeoVideo', /\$VIMEO(?:\[(.*?)\])?\((.*?)\)\$VIMEO/m) do |title, vimeo_id|
+    embed_url = %(https://player.vimeo.com/video/#{vimeo_id}?enablejsapi=1&amp;origin=#{ENV['DOMAIN']})
+    optional_title = title ? "Video: #{title}" : ''
+    %(<h2 class="govuk-heading-m govuk-!-margin-top-2">#{optional_title}</h2><div class="govspeak-embed-container" style="padding:56.19% 0 0 0;position:relative;"><iframe class="govspeak-embed-video" style="position:absolute;top:0;left:0;width:100%;height:100%;" src="#{embed_url}" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe></div>)
+  end
+
+  # TODO: Complete transcript govspeak tag
```

Comment:
I think that when we convert this to using Slim templates as we do with prompts, we'll likely want an option to render the transcript below. I can't now envisage them wanting the transcript in isolation.

Reply:
I'll remove this separate transcript tag